docs/release-notes/data-releases.md

```diff
@@ -4,20 +4,42 @@ title: Data Releases
 platform_key: DOCS_DATA_RELEASES
 ---
 
-A DCC data release is a curated data set produced from clinical and experimental data submitted through the ARGO Data Platform. Data releases happen on average four times a year and are cumulative in nature.
+A DCC data release is a curated data set produced from clinical and molecular data submitted to the ARGO Data Platform. Data releases happen on average four times a year and are cumulative in nature. Released data can be examined using the [File Repository](https://platform.icgc-argo.org/repository) and [downloaded](/docs/data-access/data-download) using the download client, provided that access to controlled data has been confirmed. To access controlled data, please see the [DACO application process here](/docs/data-access/data-access), or [apply for DACO approval here](https://icgc.org/daco).
```

Comment:
"browsed" instead of "examined"?

Reply:
i liked examined .. bc it felt like  a deeper look than browse

Reply:
ok just thought examined sounded like they were "checking" the data